platform/core/src/studies/services/wado/retrieveMetadataLoaderAsync.js

```diff
@@ -66,12 +66,16 @@ function makeSeriesAsyncLoader(
 export default class RetrieveMetadataLoaderAsync extends RetrieveMetadataLoader {
   configLoad() {
     const { server } = this;
-
-    const client = new api.DICOMwebClient({
-      url: server.qidoRoot,
-      headers: DICOMWeb.getAuthorizationHeader(server),
-    });
-
+	
+	
+	var options = {
+		url: server.baseUrl,
+        headers: DICOMWeb.getAuthorizationHeader(server),
+		qidoURLPrefix: server.qidoURLPrefix,
+		wadoURLPrefix: server.wadoURLPrefix,
+	};
```

Comment:
Is this in line with the options DICOMwebClient expects?

We should probably preserve the old configuration if `qidoURLPrefix`, `wadoURLPrefix`, and `baseUrl` are not specified. If we don't, this becomes a major/breaking change.

Mixed casing is a bit confusing: `qidoURLPrefix`, but `url` in `baseUrl` is not capitalized?

Note: incorrect indentation and some lint rules not applied.